dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -424,7 +424,7 @@ public void handleLabels(String entryPath, Set<String> addsSet, Set<String> remo
     }
 
     @Override
-    protected void handleVerifyUnverify(String entry, String verifySource, boolean unverifyRequest, boolean isScript) {
+    protected void handleVerifyUnverify(String entry, String versionName, String verifySource, boolean unverifyRequest, boolean isScript) {
         String action = "verify";
```

Comment:
If this is only being used for the help text, simplify it and move it closer. 
e.g. `String action = unverifyRequest ? "unverify" : "verify"` on line 469